templates/encounter/forms/navbar.html.twig

```diff
@@ -18,8 +18,15 @@
                         </a>
                         <div class="dropdown-menu" aria-labelledby="category_{{ category|attr }}">
                             {% for item in details.children %}
-                                {% set formURL = rootdir ~ "/patient_file/encounter/load_form.php?formname=" ~ item.directory|url_encode %}
-                                <a href="#" class="dropdown-item" onclick="openNewForm({{ formURL|attr_js }}, {{ item.displayText|xlFormTitle|attr_js }})">{{ item.displayText|xlFormTitle|text }}</a>
+                                {% if item.onclick is defined %}
+                                    {% set onclick = item.onclick %}
+                                {% else %}
+                                    {% set formURL = rootdir ~ "/patient_file/encounter/load_form.php?formname=" ~ item.directory|url_encode %}
+                                    {% set onclick = "openNewForm(" ~ formURL|attr_js ~ "," ~ item.displayText|xlFormTitle|attr_js ~ ")" %}
+                                {% endif %}
+
+                                {# <a href="#" class="dropdown-item" onclick="openNewForm({{ formURL|attr_js }}, {{ item.displayText|xlFormTitle|attr_js }})">{{ item.displayText|xlFormTitle|text }}</a> #}
```

Comment:
@robertdown If above line is meant to be commented out, then would instead just remove it. Otherwise code looks good and would then bring in if testing well for you.